DataAccess/Interfaces/IDbAccesser.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Text;
+
+namespace DataAccess.Interfaces
+{
+    public interface IDbAccesser<T>
```

Comment:
Either specify access modifiers everywhere or skip them at all (If they don't need as well.)
In some interfaces, you have a 'public' keyword before method declaration, and in others - does not.